packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -74,9 +74,13 @@ export async function getUserToInviteOrThrowIfExists({
   isOrg?: boolean;
 }) {
   // Check if user exists in ORG or exists all together
+
+  const orgWhere = isOrg && {
```

Comment:
I am not sure how this change fixes the issue.
Also, why do we need both `isOrg` and `orgId`. Shouldn't `orgId` be enough?